TG.Backend/Controllers/VehicleController.cs

```diff
@@ -0,0 +1,41 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using TG.Backend.Features.Vehicle.CreateVehicle;
+using TG.Backend.Features.Vehicle.GetVehicles;
+
+namespace TG.Backend.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+[Authorize(AuthenticationSchemes = "Bearer")]
+public class VehicleController : ControllerBase
+{
+    private readonly ISender _sender;
+
+    public VehicleController(ISender sender)
+    {
+        _sender = sender;
+    }
+
+    [HttpGet]
+    public async Task<IActionResult> GetVehicles()
+    {
+        var vehicleResponse = await _sender.Send(new GetVehiclesQuery());
+
+        if (vehicleResponse is { IsSuccess: true, StatusCode: HttpStatusCode.OK })
+            return Ok(vehicleResponse.Vehicles);
+
+        return StatusCode(503);
```

Comment:
Technically it is totally correct but I will use value from the enum here (even though we need the cast), something like that would be fine imo:
<img width="388" alt="image" src="https://user-images.githubusercontent.com/48764491/231883434-e28a7a98-0d04-400d-916c-61f5e0caec0a.png">

Reply:
You'are right, my way of doing it is not perfect :D